frontend/appflowy_flutter/lib/plugins/database_view/widgets/card/card.dart

```diff
@@ -143,29 +140,27 @@ class _RowCardState<T> extends State<RowCard<T>> {
               ),
             );
           }
-          // desktop
+
           return AppFlowyPopover(
             controller: popoverController,
             triggerActions: PopoverTriggerFlags.none,
             constraints: BoxConstraints.loose(const Size(140, 200)),
-            margin: const EdgeInsets.all(6),
             direction: PopoverDirection.rightWithCenterAligned,
-            popupBuilder: (popoverContext) => _handlePopoverBuilder(
-              context,
-              popoverContext,
-            ),
+            popupBuilder: (popoverContext) {
+              return RowActions(
+                viewId: context.read<CardBloc>().viewId,
+                rowId: context.read<CardBloc>().rowMeta.id,
```

Comment:
We already have the cardbloc no?
```suggestion
                viewId: _cardBloc.viewId,
                rowId: _cardBloc.rowMeta.id,
```